tests/MongoDB.Driver.Legacy.Tests/Builders/IndexKeysBuilderTypedTests.cs

```diff
@@ -89,7 +89,14 @@ public void CreateIndex_with_wildcardProjection_should_create_expected_index()
             var indexes = collection.GetIndexes();
             var index = indexes.RawDocuments.Single(i => i["name"].AsString == "custom");
             index["key"]["$**"].AsInt32.Should().Be(1);
-            index["wildcardProjection"].ToBsonDocument().Should().Be(BsonDocument.Parse("{ b : 1, _id : 0 }"));
+            if (CoreTestConfiguration.ServerVersion >= new SemanticVersion(4, 5, 0, ""))
+            {
+                index["wildcardProjection"].Should().Be(BsonDocument.Parse("{ b : true, _id : false }"));
```

Comment:
nit: `index["wildcardProjection"]` this part also can be moved to the line before `if`

Reply:
nit2: of even we can have code like this:

    BsonValue expectedValue = cond ? (BsonValue)true: (BsonValue)1; 
    index["wildcardProjection"].Should().Be(BsonDocument.Parse("{ b : true, _id : {expectedValue} }"));

Reply:
I'd say that separate lines look easier to read, so I would rather leave them as is.